site/themes/carvel/layouts/index.redirects

```diff
@@ -0,0 +1,4 @@
+{{ $latest   := site.Params.latest }}
```

Comment:
I think you might need to identify the multiple projects here.
Can you try changing this to something like:

```
 {{ $latest-ytt   := site.Params.ytt.version_latest }}
 {{ $latest-imgpkg   := site.Params.imgpkg.version_latest }}

/ytt/docs/         /ytt/docs/{{ $latest-ytt }}
/imgpkgdocs/         /ytt/docs/{{ $latest-ytt }}
```